tests/foreman/rhai/test_rhai.py

```diff
@@ -26,61 +27,32 @@ def test_rhai_client_setup(self):
             name='insights_{0}'.format(gen_string('alpha', 6))
         ).create()
 
-        # Create new lifecycle environment
-        lifecycle_env = entities.LifecycleEnvironment(
-            name='rhai_env_{0}'.format(gen_string('alpha', 6)),
-            organization=org,
-        ).create()
-
         # Upload manifest
         org.upload_manifest(path=manifests.clone())
 
-        # Create a custom product for RHAI
-        custom_product = entities.Product(
-            name='rhai_product_{0}'.format(gen_string('alpha', 6)),
-            organization=org,
-        ).create()
-
-        # Add insights repo to the custom product
-        custom_repo = entities.Repository(
-            content_type=u'yum',
-            name=u'insights_repo_{0}'.format(gen_string('alpha', 6)),
-            product=custom_product,
-            url=u'http://access.redhat.com/insights/repo/6/',
-        ).create()
-
-        # Sync custom repository
-        custom_repo.sync()
-
-        # Create content view
-        content_view = entities.ContentView(
-            name='rhai_cv_{0}'.format(gen_string('alpha', 6)),
-            organization=org,
-        ).create()
-
-        # Associate custom repository to new content view
-        content_view.set_repository_ids([custom_repo.id])
-
-        # Publish content view
-        content_view.publish()
-
-        # Promote content view to lifecycle_env
-        content_view = content_view.read()
-        self.assertEqual(len(content_view.version), 1)
-        content_view.version[0].promote(lifecycle_env.id)
+        # Fetching the default environment and Content View id's to be used
+        # while creating activation keys
+        default_env = org.read_json()['library_id']
+        default_cv = org.read_json()['default_content_view_id']
```

Comment:
Do organizations now have a `default_content_view_id` attribute? If so, let's update NailGun instead of just hacking around its limitations. Robottelo currently fetches NailGun directly from git, so you don't need to wait long for updates. That done, we can drop this line of code...